examples/scaling-criteo/02-ETL-with-NVTabular.ipynb

```diff
@@ -351,49 +317,6 @@
     "That's it. We initalized our Dask cluster and NVTabular will execute the workflow on multiple GPUs. Similar, we could define a cluster with multiple nodes."
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "#### Initilize Memory Pools\n",
-    "\n",
-    "Since allocating memory is often a performance bottleneck, it is usually a good idea to initialize a memory pool on each of our workers. When using a distributed cluster, we must use the `client.run` utility to make sure a function is exectuted on all available workers."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "collapsed": false,
-    "jupyter": {
-     "outputs_hidden": false
-    }
-   },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "{'tcp://127.0.0.1:37259': None}"
-      ]
-     },
-     "execution_count": 8,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "# Initialize RMM pool on ALL workers\n",
-    "def _rmm_pool():\n",
-    "    rmm.reinitialize(\n",
-    "        # RMM may require the pool size to be a multiple of 256.\n",
-    "        pool_allocator=True,\n",
-    "        initial_pool_size=(device_pool_size // 256) * 256,  # Use default size\n",
-    "    )\n",
-    "\n",
-    "\n",
-    "client.run(_rmm_pool)"
-   ]
-  },
```

Comment:
Did you mean to delete this section ?

Reply:
He addeed the `rmm_pool_size` argument to `LocalCUDACluster`, which makes this unncessary.  I'm not sure if we still need a sentence explaining that the rmm pool is important.  Perhaps we do, since we are still not using a "default" setting.